src/components/change-wallet/AddressRow.tsx

```diff
@@ -161,7 +161,7 @@ export default function AddressRow({
   );
 
   const displayAddress = useMemo(
-    () => abbreviations.address(toChecksumAddress(address) || '', 4, 6),
```

Comment:
I can't think of an instance where the checksum would fail other than address being undefined.. is this to handle ENS? wondering if we should still have`|| '` at the end but will defer

Reply:
it was showing empty strings rather than a wallet address for some reason